Broken Links Found: Polkadot

### 🚨 Link Checker Report
  **Project:** Polkadot
  **Date:** 2026-05-26
  **Workflow Run:** https://github.com/papermoonio/workflows/actions/runs/26449912716

  ---
  # Summary

| Status        | Count |
|---------------|-------|
| 🔍 Total      | 32026 |
| ✅ Successful | 31416 |
| ⏳ Timeouts   | 0     |
| 🔀 Redirected | 0     |
| 👻 Excluded   | 606   |
| ❓ Unknown    | 0     |
| 🚫 Errors     | 2     |

## Errors per input

### Errors in polkadot-mkdocs/site/smart-contracts/for-eth-devs/accounts/index.html

* [404] [https://paritytech.github.io/polkadot-sdk/master/pallet_revive/fn.is_eth_derived.html](https://paritytech.github.io/polkadot-sdk/master/pallet_revive/fn.is_eth_derived.html) | Network error: Not Found

### Errors in polkadot-mkdocs/site/ai-resources/index.html

* [ERROR] [https://help.openai.com/en/articles/12584461-developer-mode-and-mcp-apps-in-chatgpt-beta](https://help.openai.com/en/articles/12584461-developer-mode-and-mcp-apps-in-chatgpt-beta) | Network error: error sending request for url (https://help.openai.com/en/articles/12584461-developer-mode-and-mcp-apps-in-chatgpt-beta) Maybe a certificate error?
[Full Github Actions output](https://github.com/papermoonio/workflows/actions/runs/26449912716?check_suite_focus=true)

--- a/smart-contracts/for-eth-devs/accounts.md
+++ b/smart-contracts/for-eth-devs/accounts.md
@@ -55,7 +55,7 @@ The [`AccountId32Mapper`](https://paritytech.github.io/polkadot-sdk/master/palle
 
 The conversion from 32-byte Polkadot accounts to 20-byte Ethereum addresses is more complex than the reverse direction due to the lossy nature of the conversion. The [`AccountId32Mapper`](https://paritytech.github.io/polkadot-sdk/master/pallet_revive/struct.AccountId32Mapper.html){target=\_blank} handles this through two distinct approaches:
 
-- **For Ethereum-derived accounts**: The system uses the [`is_eth_derived`](https://paritytech.github.io/polkadot-sdk/master/pallet_revive/fn.is_eth_derived.html){target=\_blank} function to detect accounts that were originally Ethereum addresses (identified by the `0xEE` suffix pattern). For these accounts, the conversion strips the last 12 bytes to recover the original 20-byte Ethereum address.
+- **For Ethereum-derived accounts**: The system uses the [`is_eth_derived`](https://paritytech.github.io/polkadot-sdk/master/pallet_revive/trait.AddressMapper.html#tymethod.is_eth_derived){target=\_blank} function to detect accounts that were originally Ethereum addresses (identified by the `0xEE` suffix pattern). For these accounts, the conversion strips the last 12 bytes to recover the original 20-byte Ethereum address.
 
 - **For native Polkadot accounts**: Since these accounts utilize the whole 32-byte space and weren't derived from Ethereum addresses, direct truncation would result in lost information. Instead, the system:
 

